### PR TITLE
Floydwarshall Algorithm in c++

### PR DESCRIPTION
**Pull request by Nishtha3512**
Floyd warshall Algorithm in c++ issue #159

### CODE REVIEW

**Comment by jainaman224** [comment]
@Nishtha3512 Please complete the PR template.

File: Shortest Path algorithm [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#include<iostream>
```
Comment (jainaman224) [inline Shortest Path algorithm]: Incorrect file extension and add this to a folder.
Name of algorithm is required.

File: Shortest Path algorithm [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+         cout << setw(3) << cost[i][j];
+      cout << endl;
+   }
+}
+
+int main() {
+   floydWarshal();
+}
+
+
```
Comment (jainaman224) [inline Shortest Path algorithm]: Add sample input and output.

File: Shortest Path algorithm [hunk 3]
```diff
@@ -0,0 +1,44 @@
+#include<iostream>
+#include<iomanip>
+#define NODE 7
+#define INF 999
+using namespace std;
+
+//Cost matrix of the graph
```
Comment (jainaman224) [inline Shortest Path algorithm]: Add space after `//`.

File: Shortest Path algorithm [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+int costMat[NODE][NODE] = {
```
Comment (jainaman224) [inline Shortest Path algorithm]: Take input from user and pass array through the function instead using a global variable.

**Comment by SakshiShreya** [comment]
@Nishtha3512  are you working on this?

**Comment by somya-kapoor** [comment]
@Nishtha3512 are you working on this?

**Comment by Nishtha3512** [comment]
> @Nishtha3512 are you working on this?


,

yes, @somya-kapoor  @SakshiShreya can you please tell me how to make changes in this file?

File: Shortest Path algorithm (hunk 1 above)
Comment (somya-kapoor) [inline Shortest Path algorithm]: Please change the file name

File: Shortest Path algorithm (hunk 2 above)
Comment (somya-kapoor) [inline Shortest Path algorithm]: Add sample

File: Shortest Path algorithm [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+   {0, 3, 6, INF, INF, 3, INF},
```
Comment (somya-kapoor) [inline Shortest Path algorithm]: Take the input from user.

**Comment by somya-kapoor** [comment]
@Nishtha3512 Are you working on it?

**Comment by somya-kapoor** [comment]
@Nishtha3512 Are you working on it?

**Comment by somya-kapoor** [comment]
Due to no activity for more than 7 days, I am closing this issue.

**Pull request closed without merging**